app/component/TransitLeg.js

```diff
@@ -393,13 +394,67 @@ class TransitLeg extends React.Component {
                   color={leg.route ? `#${leg.route.color}` : 'currentColor'}
                   text={leg.route && leg.route.shortName}
                   realtime={false}
+                  icon={
+                    leg.route.shortName.startsWith('RT')
```

Comment:
@leonardehrenfried Can you think of any other possibility to check whether a leg is of type "Ruftaxi"? Mode is always bus for Ruftaxi.

Reply:
In theory you there is `route.type` which can have lots of values depending on the specific type: https://developers.google.com/transit/gtfs/reference/extended-route-types

I'm not sure if we are setting this. @derhuerst can you clarify?

Reply:
It seems that type 1500 (Taxi service) would be the most appropriate route type.

Reply:
I'm not finding a place where the route type is changed from bus (3) to something else: https://github.com/derhuerst/generate-herrenberg-gtfs-flex/search?q=type